docs/dev/jobs-config-hail.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
This file is pretty similar to `docs/dev/jobs-config.json`. If we anticipate needing multiple different config files, it'd be cleaner if we had a minimal base JSON on which all configs are built, and diff JSONs that modify the base on an ad hoc basis.

Reply:
I think this makes sense if we were just changing variables such as the `spark_version`, but there are complicated changes to the json for Hail and VEP, such as new init scripts and docker containers. 

Also a user may just want to copy the json directly to use for their own cluster configuration. 

My preference is to add an arg to `run-nb-test.py` script that points to the json

WDYT?

Reply:
added this json and updated the script

Reply:
tests passed first time! I'm getting the hang of this
@karenfeng let me know what you think